config/deployments/README.md

```diff
@@ -0,0 +1,156 @@
+# Configs for deployments
+
+## Introduction
+
+This directory contains the configs for deployments of hyperswitch in different hosted environments.
```

Comment:
Please ensure you follow similar case convention throughout the document: either `Hyperswitch` or `hyperswitch`.